### ISSUE TITLE
Masonry not working with Contao 4.6.12

### ISSUE THREAD

**Issue by coolcontent** [pdir/social-feed-bundle#7]
On my setup I have assigned Masonry with 3 columns and activated Masonry in the module. 
But only one column appears.
I tried several settings, all shows 1 column.

It's assigned to the div, but without success:
`<div class="social_feed_container masonry columns3" style="position: relative; height: 5248px;">`

In previous version (4.5.x) I didn't have this problem.

**Comment by seibtph** [comment]
I have tested this in contao 4.6.12 and it works. Do you selected both templates in the news list: **news_social_feed** and **mod_newslist_social_feed**?

Now we have released the new version 2.2.1 for the Social Feed Bundle.

**Comment by SimonSpi** [comment]
I see a similar issue on my installations as well. However the issue here is that the div tag with class "social_feed_container" does not show the masonry and columns3 classes. Interestingly I have one installation where Masonry works correctly and a duplicate installation (actually the same site I'am maintaining for testing) it is not working. Unfortunately I was not able to track down the root cause so far. Will post once I have additional info.
Btw.: Templates are set as suggested by seibtph.

**Comment by SimonSpi** [comment]
I once again have been looking into it. What I now found is that also the corresponding JavaScript files are not loaded. This means that sfMasonry is (for whatever reason) not correctly evaluated in the script. Unfortunately I still have no exact explanation what happens.
In contrast to what I have written in my previous post, I had to update the working installation from Contao 4.6.12 to 4.6.14 and I now see the issue on both of the installations.
